das_client/app/lib/pages/journey/train_journey/widgets/table/service_point_row.dart

```diff
@@ -104,6 +162,51 @@ class ServicePointRow extends CellRowBuilder<ServicePoint> {
         isRouteEnd: metadata.routeEnd == data,
         isStopOnRequest: !data.mandatoryStop,
         chevronAnimationData: config.chevronAnimationData,
+        chevronPosition: data.chevronPosition,
+      ),
+    );
+  }
+
+  @override
+  DASTableCell iconsCell1(BuildContext context) {
+    if (data.mandatoryStop && data.stationSign1 == null && data.stationSign2 == null) return DASTableCell.empty();
+
+    return _wrapToBaseHeight(
+      DASTableCell(
+        alignment: Alignment.bottomRight,
+        padding: EdgeInsets.symmetric(vertical: sbbDefaultSpacing * 0.5, horizontal: 2),
+        child: Padding(
+          padding: config.bracketStationRenderData != null ? const EdgeInsets.only(right: 24.0) : EdgeInsets.zero,
+          child: Wrap(
+            spacing: 2,
+            children: [
+              if (!data.mandatoryStop)
+                SvgPicture.asset(
```

Comment:
Nitpick: Dieses Icon mit dem colorFilter wird 5x verwendet. Evtl. auslagern in private Methode? z.B. `_icon(key: stopOnRequestKey, icon: AppAssets.iconStopOnRequest)`